src/main/scala/org/broadinstitute/dsde/workbench/sam/model/api/SamJsonSupport.scala

```diff
@@ -111,6 +112,8 @@ object SamJsonSupport {
 
   implicit val ManagedGroupAccessInstructionsFormat = ValueObjectFormat(ManagedGroupAccessInstructions.apply)
 
+  implicit val ManagedGroupSupportSummaryFormat = jsonFormat4(ManagedGroupSupportSummary.apply)
```

Comment:
If you end up making a ManagedGroup file for the model, the json formatting could go in there as well in the companion object.